### DISCUSSION TITLE
I use the incorrect mail address to register the account.

### DISCUSSION THREAD

**Discussion by lxfater** [npm/feedback#664] [Website] [answered]
I use the incorrect mail address to register the account. Now I can not log in to the account even though I have the password since It needs a one-time password to log in.  Please help! I make a mistake to type the ".com" to ".cpm". It would be helpful for me to get this account back since I very much like the username. Can somebody help me?

I think the website lack some procedure to check the email address before registering an account.

**Comment by monishcm** [comment, marked as answer]
please contact support!

**Comment by AurelianSpodarec** [comment]
I got the same issue
